uaclient/cli.py

```diff
@@ -1452,6 +1452,9 @@ def action_auto_attach(args, *, cfg: config.UAConfig) -> int:
     except exceptions.UrlError:
         event.info(messages.ATTACH_FAILURE.msg)
         return 1
+    except exceptions.EntitlementsNotEnabledError as e:
```

Comment:
sorry I could have explained my thought better
I think this part of the code is not needed. Defining the `exit_code` for `UserFacingError` is enough for the main error handler to exit correctly ( like happens today with the `AlreadyAttachedError`, for instance)

Reply:
Fair enough, good point. I will update the code

Reply:
@renanrodrigo fixed